test/PSMVariant2Actions.t.sol

```diff
@@ -0,0 +1,575 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { MockERC20 } from "lib/erc20-helpers/src/MockERC20.sol";
+
+import { ERC4626Mock }     from "./mocks/ERC4626Mock.sol";
+import { PSMVariant2Mock } from "./mocks/PSMVariant2Mock.sol";
+
+import { PSMVariant1Actions } from "src/PSMVariant1Actions.sol";
+
+// NOTE: This test contract is demonstrate support for the second version of the PSM with the version 1 actions
+abstract contract PSMVariant2ActionsBase is Test {
```

Comment:
```suggestion
// NOTE: This test contract demonstrates support for the second version of the PSM with the
//       version 1 actions. For this reason, all of the code is identical to
//       test/PSMVariant1Actions.t.sol, except for the setup and the redundant testing of the
//       PSMVariant1Actions constructor. This can be seen by diffing the two files.
```